src/app-layout/index.tsx

```diff
@@ -197,6 +197,14 @@ const OldAppLayout = React.forwardRef(
       }
     };
 
+    useEffect(() => {
+      // Close navigation drawer on mobile so that the main content is visible
+      if (isMobile) {
+        onNavigationToggle(false);
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [isMobile]);
```

Comment:
As opposed to adding more `eslint-disable-next-line` exceptions, I suggest converting `const onNavigationToggle = useCallback(() => {}, [])` to `const onNavigationToggle = useStableCallback(() => {})`. This value will be safe to put into effects